custom/icds_reports/static/js/directives/cas-export/cas-export.directive.js

```diff
@@ -74,6 +76,8 @@ function CasExportController($window, $location, locationHierarchy, locationsSer
 
     vm.onSelectYear = function (year) {
         var latest = new Date();
+        var offset = latest.getDate() < 15 ? 2 : 1;
+        latest.setMonth(latest.getMonth() - offset);
```

Comment:
what happens if `latest.getMonth()` returns a number less than the offset. For example in before 15 January

Reply:
It's going to change the year.